Sources/Networking/Operations/Handling/CustomerInfoResponseHandler.swift

```diff
@@ -26,7 +26,7 @@ class CustomerInfoResponseHandler {
         purchasedProductsFetcher: PurchasedProductsFetcherType,
         productEntitlementMapping: ProductEntitlementMapping?,
         userID: String,
-        offlineEntitlementsEnabled: Bool = false
+        offlineEntitlementsEnabled: Bool = true
```

Comment:
⚠️  This PR enables offline entitlements (so we can test them on integration tests).
I think instead I'll get rid of the flag?